src/Sentry/Internal/Hub.cs

```diff
@@ -109,41 +108,28 @@ public void WithScope(Action<Scope> scopeCallback)
         {
             var transaction = new Transaction(this, context);
 
-            // Transactions are not handled by event processors, so some things need to be added manually
-
-            // Make a sampling decision if it hasn't been made already.
-            // It could have been made by this point if the transaction was started
-            // from a trace header which contains a sampling decision.
-            if (transaction.IsSampled is null)
+            // Dynamic sampling is ran regardless of whether a decision
```

Comment:
Lets not call this "dynamic sampling" because there's a feature in the works called Dynamic Sampling and it involved being able to change sampling on the server via configuration that affects Relays ingesting the data.

This is the trace sampler callback.